src/index.scss

```diff
@@ -1,6 +1,5 @@
 // index.scss is for CSS declarations that aren't specific to a component.
 // In practice, that means this file is used for two things:
-//
```

Comment:
New `stylelint` rule doesn't want blank comments like this, I sorta disagree, but followed the rule to keep things as vanilla as possible. Should we disable this?

Reply:
I agree, this rule seems a bit much. I vote to disable the rule.

Reply:
✅ disabled!